README.md

```diff
@@ -8,8 +8,11 @@ at the heart of the [Wikibase software](http://wikiba.se/).
 
 ## Release notes
 
-### 4.0.0 (dev)
+### 4.0.0 (2017-10-09)
 
+* Made the library a pure JavaScript library.
+* Removed MediaWiki ResourceLoader module definitions.
+* Raised DataValues JS library version requirement to 0.10.0.
```

Comment:
Is it really necessary to drop the compatibility with DataValues JavaScript 0.9?

Also, I think we should use the exact name of the component: "DataValues JavaScript".

Reply:
It is necessary in my eyes, given dropping all php stuff and depending on the npm package (which only exists for 0.10).

Re names: well I can change it sure, but: the name you gave is neither name of the old packagist package nor the name of the npm package. So how it is the "exact name of the component"?

Reply:
It's ["DataValues JavaScript"](https://github.com/wmde/DataValuesJavaScript/blob/master/README.md), never "JS".

I think I understand why this isn't compatible with 0.9 any more. Thanks.